deploy/deploy_backend.sh

```diff
@@ -1,23 +1,28 @@
 #!/usr/bin/env bash
 ssh -o "StrictHostKeyChecking=no" -i deploy_rsa -A ${DEPLOY_USERNAME}@${DEPLOY_HOST} << EOT
 become ${DEPLOY_BACKEND}
-source ~/www/python/venv/bin/activate
-echo "Activated the environment"
 cd ~/www/python/
 git clone https://github.com/wikimedia/WikiContrib/
+echo "Cloned the repository"
 mv src/WikiContrib/.env WikiContrib/backend/WikiContrib/WikiContrib/.env
 cd src/
 rm -rf WikiContrib/ Install.md manage.py query/ result/ test_data/ requirements.txt db.sqlite3
 cp -r ../WikiContrib/backend/WikiContrib/* .
 cd WikiContrib
 mv local_settings.py local_settings_sample.py
 cd ../
+webservice --backend=kubernetes python3.7 shell
+echo "Activated kubernetes webservice shell"
+source ~/www/python/venv/bin/activate
+pip install --upgrade pip wheel
```

Comment:
I don't think this is needed verytime we restart the server.. Can you check it out ?

Reply:
It's actually necessary @rammanoj . We aren't running those commands in the past and what was the reason our backend always breaks when we merge. Whenever I go to fix those breakages, the above commands are what I use to fix it

Reply:
Not asking about all the above commands. I am asking about only one command "pip install --upgrade pip wheel"

Reply:
Might look unnecessary, but it is part of the standard set of commands outlined in the documentation. It wouldn't be in the documentation if it wasn't important. It might not be necessary if we just recently updated pip, but if we remove that command, things might break in the future if the version of pip running becomes too old. It is always advisable to update pip before running pip Install (at least in all Linux courses I have taken and Google cloud courses)